content/applications/productivity/iot/devices/camera.rst

```diff
@@ -55,8 +55,8 @@ is reached during the manufacturing process, the database prompts the operator t
    to create a new quality check from the :guilabel:`Quality Checks` page.
 
 .. seealso::
-   - :doc:`../../../inventory_and_mrp/manufacturing/quality_control/quality_control_points`
-   - :doc:`../../../inventory_and_mrp/manufacturing/quality_control/quality_alerts`
+   - :doc:`../../../inventory_and_mrp/quality/quality_management/quality_control_points`
+   - :doc:`../../../inventory_and_mrp/quality/quality_management/quality_alerts`
```

Comment:
I was told by Sam that when linking to docs, we want to use absolute paths, like the `/applications/....` format that we used in this [Documentation guideline](https://www.notion.so/Documentation-Guidelines-78bc0a486181431ebab8d4145064eafa?pvs=4#d078e791906f4f87aca3ab6b8c6bce05). Not to mansplain to you, but what's used here is a [relative path](https://www.codingrooms.com/blog/file-paths#:~:text=A%20relative%20path%20describes%20the,regularly%20use%20relative%20file%20paths.)! 
```suggestion
   - :doc:`Quality control points
     </applications/inventory_and_mrp/quality/quality_management/quality_control_points>`
   - :doc:`Quality alerts
     </applications/inventory_and_mrp/quality/quality_management/quality_alerts>`
```
Also in my suggestion, I added a line because the whole thing would've exceeded 100 chars for the line!